Add testing instructions for payment gateway suggestions

joshuatf opened the pull request:
Fixes (part of) woocommerce/woocommerce-admin#7039 

Testing instructions for new payment gateway suggestions.

### Detailed test instructions:

Make sure that these testing instructions are clear and easy to follow and cover most use cases without being too specific.

joshuatf committed:
Add testing instructions for payment gateway suggestions

louwie17 commented on TESTING-INSTRUCTIONS.md:
```diff
@@ -2,6 +2,107 @@
 
+### Payment gateway suggestions feature
+
+1. Note that the `payment-gateway-suggestion` feature should be turned on in your current build
```
How would someone from GlobalStep check this exactly? Is there a specific zip file we have that has it enabled (that we can provide)?

joshuatf replied:
Good question.  Initially I made these test instructions with Isotope in mind, but I do think it will go to GlobalStep eventually so I'll remove this.

This will be turned on in all environments as of https://github.com/woocommerce/woocommerce-admin/pull/7224

louwie17 commented on TESTING-INSTRUCTIONS.md:
```diff
@@ -2,6 +2,107 @@
+2. Note that the gateway is installed if it requires a plugin
+3. On the connection step, a button should be presented that links to the gateway's (legacy) settings screen
+4. Delete the plugin
+5. Clone and checkout one of the gateway PRs in this issue - https://github.com/woocommerce/woocommerce-admin/issues/6878
```
I am not sure if this is do-able by GlobalStep testing, maybe we want to provide plugin zip's of each PR mentioned in #6878 (we could just upload them to the PR thread)?

joshuatf replied:
That's a good idea.  I'll update this and make an issue to collect those plugins.

louwie17 commented on TESTING-INSTRUCTIONS.md:
```diff
@@ -2,6 +2,107 @@
+1. Set your store country to any country except India
+2. Don't select CBD as an industry during onboarding
+3. Make sure the PayPal connection flow is shown and works as expected
+4. Throw an error in the REST endpoint `WC_PAYPAL_NAMESPACE + '/onboarding/get-params'`
```
Instead of doing this from the code, could we use the `Network request blocking` feature in Chrome console for the `*/onboarding/get-params` url?

louwie17 reviewed (commented):
@joshuatf I left a couple comments, mostly to make the test instructions simpler, as I believe these will mostly be used by GlobalStep.

joshuatf committed:
Handle PR feedback

joshuatf committed:
Add plugin links for testing

joshuatf commented:
Thanks for the feedback, @louwie17 💯 

Updated per feedback and added some links to prebuilt test zips for plugins in https://github.com/woocommerce/woocommerce-admin/pull/7205/commits/dc2e37de891f676bdefa61a4f0d70849c4d55a12.  @joelclimbsthings @samueljseay if you have any you want to add, feel free to add them to this list in this PR or a follow-up.

louwie17 reviewed (approved):
Thanks for adding the suggestions, especially the plugin zip files, looks great!

louwie17 commented on TESTING-INSTRUCTIONS.md:
```diff
@@ -26,6 +26,113 @@ add_filter( 'woocommerce_admin_export_id', function ($export_id) {
+* PayFast - https://mc.a8c.com/includes/img-uploader/files/1624475037-woocommerce-gateway-payfast.zip
+* PayPal - https://mc.a8c.com/includes/img-uploader/files/1624475037-woocommerce-paypal-payments.zip
+* RazorPay - https://mc.a8c.com/includes/img-uploader/files/1624475037-razorpay-woocommerce.zip
+* Stripe - https://mc.a8c.com/includes/img-uploader/files/1624475040-woocommerce-gateway-stripe.zip
```
@joshuatf this is not a huge issue, but I noticed that this stripe zip version contains this as the setupHelpText:
```
PayU can be configured under your store settings. Create your PayU account here.
```
The PR for the stripe plugin looks correct.
